payments/schema/mutations.py

```diff
@@ -298,11 +298,10 @@ def mutate_and_get_payload(cls, root, info, **input):
             lease = get_node_from_global_id(
                 info, lease_id, BerthLeaseNode, nullable=True
             )
-            # TODO: Check for WinterStorageLeaseNode
-            # if not lease:
-            #     product = get_node_from_global_id(
-            #         info, product_id, WinterStorageLeaseNode, nullable=True
-            #     )
+            if not lease:
+                lease = get_node_from_global_id(
+                    info, lease_id, WinterStorageLeaseNode, nullable=True
+                )
```

Comment:
should we once again check `if not lease` and if there is no lease, raise an error? 🤔

Reply:
🤔 that would be reasonable.

Reply:
Updated with some proper checking 👍

Reply:
Updated with some proper checking 👍

Reply:
nice 👍